pkg/apis/pingcap/v1alpha1/validation/validation.go

```diff
@@ -605,3 +609,21 @@ func validateAdditionalContainers(containers []corev1.Container, fldPath *field.
 
 	return allErrs
 }
+
+func validateStorageInfo(spec *v1alpha1.TidbMonitorSpec, fldPath *field.Path) field.ErrorList {
```

Comment:
Suggest passing the `spec.Storage` as a parameter instead of `spec` to make this function generic, which can be used by validating other similar fields.